kensu/utils/kensu.py

```diff
@@ -426,22 +442,31 @@ def add_dependencies(self, ins, outs, mapping_strategy=mapping_strategies.FULL):
                 new_outs.append(o)
 
         self.dependencies.append((new_ins, new_outs, mapping_strategy))
+
+    def get_lineage_name(self,
+                         data_flow  # type: list[SchemaLineageDependencyDef]
+                         ):
+        inputs = ",".join(sorted(self.to_schema_names([d.from_schema_ref.by_guid for d in data_flow])))
```

Comment:
I think `join` throws if empty (FWIW)

Reply:
it doesn't. throws only if passed a not iterable